src/igvfd/tests/fixtures/schemas/treatment.py

```diff
@@ -2,7 +2,7 @@
 
 
 @pytest.fixture
-def treatment_1(testapp):
+def treatment(testapp):
```

Comment:
How about we make it treatment_chemical to match treatment_protein?

Reply:
Not sure longer string but more description is better treatment_lactate_1_hr? @jenjou do you think this is acceptable?

Reply:
I like `treatment_chemical`. I'm having trouble thinking of cases where we want to specifically test the treatment duration, so I think that might be too much detail to include in the fixture name.

Reply:
Okay